engine/graphics/src/opengl/graphics_opengl.cpp

```diff
@@ -1073,7 +1073,7 @@ static void LogFrameBufferError(GLenum status)
         context->m_IsGles3Version = 1; // 0 == gles 2, 1 == gles 3
         context->m_PipelineState  = GetDefaultPipelineState();
 
-#if defined(__EMSCRIPTEN__) || defined(__ANDROID__)
+#if defined(__EMSCRIPTEN__) || defined(__ANDROID__) || (defined(__linux__) && defined(__aarch64__))
```

Comment:
These switches start to get complicated.
I propose moving this switch to the `wscript`, and adding a `DM_GRAPHICS_USE_OPENGL`/`DM_GRAPHICS_USE_OPENGLES` respectively.

Reply:
Agreed.